root/components/home/Intro.js

```diff
@@ -0,0 +1,265 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {Modal} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faSearch, faScanner} from '@fortawesome/free-solid-svg-icons';
+
+import {reduceArtistCredit}
+  from '../../static/scripts/common/immutable-entities';
+import {ArtworkImage} from '../Artwork';
+import entityHref
+  from '../../static/scripts/common/utility/entityHref';
+
+import Blog from './Blog';
+
+const chipData = [
+  {key: 0, label: 'Artist'},
+  {key: 1, label: 'Release'},
+  {key: 2, label: 'Recording'},
+  {key: 3, label: 'Label'},
+  {key: 4, label: 'Work'},
+  {key: 5, label: 'Release Group'},
+  {key: 6, label: 'Area'},
+  {key: 7, label: 'Place'},
+  {key: 8, label: 'Annotation'},
+  {key: 9, label: 'CD Stud'},
+  {key: 10, label: 'Editor'},
+  {key: 11, label: 'Tag'},
+  {key: 12, label: 'Instrument'},
+  {key: 13, label: 'Series'},
+  {key: 14, label: 'Event'},
+  {key: 15, label: 'Documentation'},
+];
+
+let typeCurrent = 'Artist';
+
+export default class Intro extends React.Component {
+    state = {
+      data: 'Actively looking for a barcode...',
+      show: false,
+    };
+
+    handleClose = () => {
+      this.setState({show: false});
+    }
+
+    handleShow = () => {
+      this.setState({show: true});
+    }
+
+    render() {
+      return (
+        <section className="intro d-flex align-items-center" id="intro">
+          <div className="container">
+            <div className="row">
+              <div
+                className="col-lg-9 d-flex flex-column"
+              >
+                <h1
+                  className="ms-4 mt-4"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l('The Music Database')}
+                </h1>
+                <h2
+                  className="ms-4 mt-2"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l(`World's Biggest Open Source Music Database`)}
```

Comment:
Only claim this if you're certain. I'm not, but I also haven't done the research.

Reply:
I'm not sure whether there is a downside to claiming. Almost every company I see claims that they are the biggest. Tbh I do believe (without exhaustive research) that MusicBrainz is the largest open-source music database. This statement in any case is powerful and is definitely a pick for me. (aerozol suggested this text btw! :) )